lib/generators/react/component_generator.rb

```diff
@@ -129,15 +129,23 @@ def component_name
 
       def file_header
         if webpacker?
-          %|var React = require("react")\n|
+          if options[:es6]
```

Comment:
Is it important to distinguish between these two varieties of preprocessed JS? I mean, what's the scenario where you _want_ `require(...)` but not `import`? 

Anyone who's using webpacker will probably _already_ use Babel (for JSX), right? So do you think one of these options would be sufficient by itself?

Reply:
I was tempted to scrap all the old style generators but was not sure on the effect it would have with sprockets + babel transpiler. I did what looked safe given I still don't fully know the system and saw the `option[:es6]` used nearby in the same file.
Will continue on this path a little this afternoon, seems like the most pressing issue for now, then maybe the HMR ( 793 ) thing after once I get time.